lib/fetchers/url.rb

```diff
@@ -178,5 +163,42 @@ def download_archive(path)
       @temp_archive_path = nil
       final_path
     end
+
+    def http_opts
+      opts = {}
+      opts[:ssl_verify_mode] = OpenSSL::SSL::VERIFY_NONE if @insecure
+
+      if @config['server_type'] == 'automate'
```

Comment:
Yeah, that makes sense to me. I didn't want to tackle that in this PR but rather just add the error checking. Some better tests need to be written when we do that, and probably before, also.